docs/pages/dictionary.mdx

```diff
@@ -0,0 +1,262 @@
+import { Callout } from "nextra/components";
+
+# Solidity to MUD Dictionary
+
+To properly use MUD you need to do some things differently than the way you do them normally in Solidity.
+
+## State variables
+
+A `System` should be stateless, for several reasons:
+
+- By concentrating all the state information in a single contract (the `World`) we make it easier to apply [access control](/world/namespaces-access-control), [execute hooks when data changes](/store/store-hooks), etc.
+- It is a lot easier to [upgrade functionality](/cli/deploy#upgrading-a-world) when you do not need to deal with storage because the contract does not have a state.
+- The same `System` can be used by multiple `World`s.
+
+The functionality of [state variables](https://docs.soliditylang.org/en/latest/internals/layout_in_storage.html) is implemented using [tables](/store/tables).
```

Comment:
```suggestion
Instead of using [state variables](https://docs.soliditylang.org/en/latest/internals/layout_in_storage.html) you can achieve the same functionality with [tables](/store/tables).
```